build.py

```diff
@@ -467,8 +467,11 @@
 # TODO: Probably this should be another 'product', but for now it's simply a phase
 foundation_tests = SwiftExecutable('TestFoundation', [
 	'TestFoundation/main.swift',
+        'TestFoundation/HTTPServer.swift',
```

Comment:
I am not sure. But ```swift_sources```  seems to be holding all the swift source files that are compiled into Foundation. ```HTTPServer.swift``` needs to be compiled into TestFoundation.